src/App/Components/Comment/comment.holder.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import { connect, useSelector } from 'react-redux';
+import CommentBody from './parent.comment.body';
+import actions from '../../../Redux/Actions/getAllComments';
+
+const {
+  getAllComments, postComment, updateComment, postCommentReply
+} = actions;
+
+const CommentHolder = ({
+  getAllComments: getComments,
+  postComment: createComment,
+  updateComment: newComment,
+  postCommentReply: replyComment,
+  slug
+}) => {
+  const data = useSelector(state => state.commentsReducer.All_Comments);
+  const userData = useSelector(state => state.notifications.profile);
+  const extractedData = userData.profile || '';
+  const [commentText, setComment] = useState('');
+
+  const submitReplies = async (slugText, replyText, id) => {
+    await replyComment(slugText, replyText, id);
+    getComments(slugText);
+  };
+
+  const submitComment = async (e) => {
+    e.preventDefault();
+    const datacamp = await createComment(slug, commentText);
+    newComment(datacamp);
+  };
+
+  useEffect(() => {
+    getComments(slug);
+  }, [getComments]);
+
+  return (
+    <div className="comment-holder mt-5">
+      <div className="comment-holder__title ml-4">
+        <h3 style={{ fontSize: '25px' }}>COMMENTS</h3>
+        <div className="parent-reply-input">
+          <a href={`/profile/${extractedData.username}`}><img className="rounded-circle comment-body__writer-image" src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png" alt="Generic placeholder" /></a>
```

Comment:
I have to change the image to be the one of the user stored in the database

Reply:
okay @MCFrank16